### DISCUSSION TITLE
CRUD operations in Brands

### DISCUSSION THREAD

**Discussion by minas96** [XiaoFaye/WooCommerce.NET#736] [Q&A]
Have you implemented interface for brands(e.g. get all, create, etc.)?
Example of url that return data https://example.gr/wp-json/wc/v3/products/brands

**Comment by XiaoFaye** [comment]
Hi, the brands api endpoint is coming from WooCommerce Brands plugins, so it will not be implemented in the standard version of WooCommerce.NET. But I'm happy to discuss if you would like to request customized REST API development for this, please email me on xiaofaye(a)msn.com

**Comment by richardaubin** [comment]
Brands is now part of core.
